topics/transcriptomics/tutorials/scrna-case_alevin-combine-datasets/tutorial.md

```diff
@@ -0,0 +1,321 @@
+>    - *"Function to manipulate the object"*: 'Concatenate along the observations axis'
+>    - {% icon param-file %} *"Annotated data matrix to add"*: 'Select all the other matrix files from bottom to top'
+>    - *"Join method"*: `Intersection of variables`
+>    - *"Key to add the batch annotation to obs"*: `batch`
+>    - *"Separator to join the existing index names with the batch category"*: `-`
+{: .hands_on}
+
+Now let's look at what we've done! Unfortunately, AnnData objects are quite complicated, so the {% icon galaxy-eye %} won't help us too much here. Instead, we're going to use a tool to look into our object from now on.
+
+> ### {% icon hands_on %} Hands-on: Inspecting AnnData Objects
+>
+> 1. {% tool [Inspect AnnData](toolshed.g2.bx.psu.edu/repos/iuc/anndata_inspect/anndata_inspect/0.7.5+galaxy1) %} with the following parameters:
+>    - {% icon param-file %} *"Annotated data matrix"*: output of **Manipulate AnnData** {% icon tool %}
+>    - *"What to inspect?"*: `General information about the object`
+> 2. {% tool [Inspect AnnData](toolshed.g2.bx.psu.edu/repos/iuc/anndata_inspect/anndata_inspect/0.7.5+galaxy1) %} with the following parameters:
+>    - {% icon param-file %} *"Annotated data matrix"*: output of **Manipulate AnnData** {% icon tool %}
+>    - *"What to inspect?"*: `Key-indexed observations annotation (obs)`
+> 3. {% tool [Inspect AnnData](toolshed.g2.bx.psu.edu/repos/iuc/anndata_inspect/anndata_inspect/0.7.5+galaxy1) %} with the following parameters:
+>    - {% icon param-file %} *"Annotated data matrix"*: output of **Manipulate AnnData** {% icon tool %}
+>    - *"What to inspect?"*: `Key-indexed annotation of variables/features (var)`
+{: .hands_on}
+
+Now have a look at the three {% icon tool %} **Inspect AnnData** outputs.
+
+> ### {% icon question %} Question
+>
+> 1. How many cells do you have now?
+> 2. Where is `batch` information stored?
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1. If you look at the **General information** {% icon tool %} output, you can see there are now `338 cells`, as the matrix is now 338 cells x 35734 genes. You can see this as well in the **obs** {% icon tool %} (cells) and **var** {% icon tool %} (genes) file sizes.
+> > 2. Under **Key-indexed observations annotation (obs)**. Different versions of the Manipulate tool will put the `batch` columns in different locations. The tool version in this course puts `batch` in the `8th` column. Batch refers to the order in which the matrices were added. The files are added from the bottom of the history upwards, so be careful how you set up your histories when running this (i.e. if your first dataset is N703 and the second is N701, the `batch` will call N703 `0` and N701 `1`!)
+> {: .solution}
+>
+{: .question}
+
+# Adding batch metadata
+
+I set up the example history with the earliest indices at the bottom.
+
+![Ordered history](../../images/scrna-casestudy/wab-history-files-ascending.png "Correct order of the history")
```

Comment:
It looks like this alt-text doesn't describe the image very well for users who are relying on a screen reader to access the content. We have a short guide on [what's appropriate for alt-text vs captions in the GTN Contributing Guide](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#guidelines-on-alt-vs-figcaption-text) which has more details.